pkg/util/injection.go

```diff
@@ -127,6 +129,20 @@ func CreateTruncateFilesScript(files []string, scriptName string) (string, error
 	return f.Name(), err
 }
 
+// CreateDeleteFilesScript creates a shell script that contains removal
+// of the provided files injected into the container. The path to the script is returned.
+func CreateDeleteFilesScript(files []string, dir string) (string, error) {
```

Comment:
any reason you ended up going with this approach instead of generating a "RUN rm x && rm y && rm z" command?

Reply:
(it's an uglier dockerfile but it seems less fiddly than generating a script, injecting it, running it, and deleting it)

Reply:
cleaner Dockerfile. Although in terms of performance, running a bunch of `rm` commands will need one less layer to achieve the same result.